tincr/io/library/genFamilyInfo.tcl

```diff
@@ -38,7 +38,10 @@ proc createBlankDesignByPart { part } {
     return [tincr::designs new mydes $part]
 }
 
-# helper function used to get the relative name of Vivado elements 
+# helper functions used to get the relative name and type of Vivado elements 
+proc prefix { s p } {
```

Comment:
These parameter names (`s` and `p`) are too vague. How about `string` and `delimiters`?